x-pack/test_serverless/api_integration/test_suites/common/alerting/summary_actions.ts

```diff
@@ -64,31 +71,44 @@ export default function ({ getService }: FtrProviderContext) {
       'kibana.alert.consecutive_matches',
     ];
 
+    before(async () => {
+      roleAdmin = await svlUserManager.createApiKeyForRole('admin');
+      internalReqHeader = svlCommonApi.getInternalRequestHeader();
+    });
+
     afterEach(async () => {
       await supertest
         .delete(`/api/actions/connector/${connectorId}`)
-        .set('kbn-xsrf', 'foo')
-        .set('x-elastic-internal-origin', 'foo')
+        .set(internalReqHeader)
+        .set(roleAdmin.apiKeyHeader)
         .expect(204);
       await supertest
         .delete(`/api/alerting/rule/${ruleId}`)
-        .set('kbn-xsrf', 'foo')
-        .set('x-elastic-internal-origin', 'foo')
+        .set(internalReqHeader)
+        .set(roleAdmin.apiKeyHeader)
         .expect(204);
```

Comment:
The combination we're running here is confusing: we run `supertest` (runs with operator role) and then we still provide the API key. We should only do one or the other.

Reply:
Prolly a regex mishap :(

Reply:
https://github.com/elastic/kibana/pull/183381/commits/2e34145b64e3d4f940ad3c393b6af527246915a8